src/screens/SiteImageGalleryScreen.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+import {StyleSheet} from 'react-native';
+import {connect} from 'react-redux';
+import Gallery from 'react-native-image-gallery';
+
+import _ from 'lodash';
+
+class SiteImageGalleryScreen extends Component {
+
+    render() {
+        const {fillScreen} = styles;
+        const {selectedSite: {siteImageData}} = this.props;
+
+        return (
+            <Gallery
+                style={{flex: 1, backgroundColor: 'black'}}
+                images={[
+                    {source: {uri: `data:image/png;base64,${siteImageData}`}}
+                ]}
+            />
```

Comment:
We'll be able to use this gallery for multiple images when we support it. For now, it lets us inspect and zoom on the single site image.